recipes/pytest-codspeed/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "pytest-codspeed" %}
+{% set version = "2.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pytest_codspeed-{{ version }}.tar.gz
+  sha256: 665003fc20117b64a98d16ffd1008f5bd6bf3b1e9af142b98c00abff7f626bbd
```

Version 2.2.1 is out now - https://github.com/CodSpeedHQ/pytest-codspeed/releases/tag/v2.2.1

```suggestion
{% set version = "2.2.1" %}

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pytest_codspeed-{{ version }}.tar.gz
  sha256: 0adc24baf01c64a6ca0a0b83b3cd704351708997e09ec086b7776c32227d4e0a
```